x/leverage/keeper/keeper_test.go

```diff
@@ -175,48 +175,6 @@ func (s *IntegrationTestSuite) TestWithdrawAsset_Valid() {
 	s.Require().Equal(int64(0), uTokenBalance.Amount.Int64())
 }
 
-func (s *IntegrationTestSuite) TestWithdrawAsset_WithExchangeRate() {
```

Comment:
Note: Removed test as manually setting exchange rates is no longer possible (they are always derived instantly)